Havit.Blazor.Components.Web.Bootstrap.Documentation/Shared/Components/Search/Search.razor.cs

```diff
@@ -111,6 +111,13 @@ private SearchItem SelectedResult
 			new("/components/HxTreeView", "HxTreeView", "hierarchy"),
 			new("/components/HxValidationMessage", "HxValidationMessage", "form"),
 
+			// Concepts
+			new("/concepts/SettingsAndDefaults", "Settings & Defaults", ""),
+			new("/concepts/Settings", "Settings", ""),
+			new("/concepts/Defaults", "Defaults", ""),
+			new("/concepts/SettingsAndDefaults#Overriding", "Settings & Defaults > Overriding", "override"),
+			new("/concepts/SettingsAndDefaults#WhereToSet", "Defaults > Where to set", ""),
+
```

Comment:
I decided to keep the search index small, these additional items add no value, please remove.